x-pack/legacy/plugins/uptime/public/components/settings/indices_form.tsx

```diff
@@ -63,7 +62,7 @@ export const IndicesForm: React.FC<SettingsFormProps> = ({
               id="xpack.uptime.sourceConfiguration.heartbeatIndicesDefaultValue"
               defaultMessage="The default value is {defaultValue}"
               values={{
-                defaultValue: <EuiCode>{defaultDynamicSettings.heartbeatIndices}</EuiCode>,
+                defaultValue: <EuiCode>{dss.settings.heartbeatIndices}</EuiCode>,
```

Comment:
This change will display the current value? default value should come from constant?

Reply:
I was supplying the default to the reducer's initial state as part of the same change, but I'll put it back now, as we've discussed why we can't use defaults for settings on the client above.

Reply:
Right you are, we don't want that to change!

Reply:
This is using defaults again now.